Tasks/AzureWebAppContainerV1/Tests/utils.ts

```diff
@@ -50,6 +50,10 @@ export function mockTaskArgument():  ma.TaskLibAnswers{
                 },
                 "webAppPkg": {
                     "isDirectory": true
```

Comment:
Why do we have changes in this file? Does this PR contain L0s as well?

Reply:
no, it was made to fix some L0, 
Existing L0 is sufficient here.